make input descriptor constraints required (#366)
decentralized-identity/presentation-exchange: spec/spec.md

  (e.g., `jwt`, `jwt_vc`, `jwt_vp`, etc.). This `format` property is identical in
  value signature to the top-level `format` object, but can be used to specifically
  constrain submission of a single input to a subset of formats or algorithms.
- The [[ref:Input Descriptor Object]] ****MUST**** contain a `constraints`

>   property. Its value ****MUST**** be an object composed as follows, with
>    _at least one property_ unless otherwise specified by a [[ref:Feature]]:

Reviewer: ```suggestion
property. Its value ****MUST**** be an object composed of _at least one_
of the following properties, unless otherwise specified by a [[ref:Feature]]:
```
